### PR TITLE
Game win condition

### LINKED ISSUE #80: End Game Conditions & Screen
Finish the End game conditions and create a UI screen for the end game

### PR DESCRIPTION
**Pull request by Mart500**
Created EndGameScript to interface with newly created EndGameScene. 
Created GameManager functions to create a scoreboard and pass it to EndGameScene.
EndGameScene should load when no tiles are left and display a primitive scoreboard.
Buttons in EndGameScene should quit the application (when it's built) and start a new game.
The Play again button, whilst working, does not appear so as main menu has not yet been created. 
closes #80

### CODE REVIEW

File: Assets/Code/Scripts/tst.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```
Comment (RobinStephenson) [inline Assets/Code/Scripts/tst.cs]: What is this file?

**Review by RobinStephenson — CHANGES_REQUESTED** [review]

File: Assets/Code/Scripts/tst.cs (hunk 1 above)
Comment (Mart500) [reply Assets/Code/Scripts/tst.cs]: A Redundant file will remove.

**Review by Mikeywalsh — CHANGES_REQUESTED** [review]
Can you fix the testing aswell? It's not a lot of work, just have it use the new method instead and check if the winner is actually the first element in the list etc

**Comment by Mart500** [comment]
@Mikeywalsh I thought I'd done that. Not sure how you'd like me to change. I'd kept the same function for testing and changed the assert to ensure that in endgamescript the correct winner is in the winning position in the list and that it's values are correct. If you can just describe how you'd like it done I can do it.

**Comment by Mikeywalsh** [comment]
@Mart500 Sorry, was just me being stupid and not seeing the changes. All looks good

**Review by Mikeywalsh — APPROVED** [review]

**Comment by Mart500** [comment]
@RobinStephenson Removed random file.

**Review by RobinStephenson — APPROVED** [review]

**Comment by Mikeywalsh** [comment]
@Mart500 I'm just going through the conflicts now. I might break GameManager since its a very old version of it that you was using, before even AI was put in the class, so i have to go through it manually and select what functions i want to keep. In future can you merge with master every time master changes to avoid this type of stuff!

**Comment by Mart500** [comment]
@Mikeywalsh Apologies, the branch is less than 2 days old, I didn't imagine it was so much of an issue. Will pull from master daily from now on

**Comment by RobinStephenson** [comment]
@Mikeywalsh do you want to check the conflicts fix if not then just merge

**Comment by Mikeywalsh** [comment]
Im testing the branch out locally, the test fails 100% of the time and scenemanager wasn't importer so it wouldn't compile anyway, please make sure that things work before submitting a pull request @Mart500

**Comment by Mikeywalsh** [comment]
Finished fixing things, everything now works, including the quit to main menu button in the game scene

**Pull request merged**